examples/c/gpio_advanced.c

```diff
@@ -16,51 +16,100 @@
 
 /* mraa header */
 #include "mraa/gpio.h"
+#include "mraa/led.h"
+
+/* GPIO default */
+#define GPIO_PIN1 5
 
-#define GPIO_PIN 6
+/* LED default */
+#define LED_NUM1 0
 
-void
-int_handler(void* args)
-{
+volatile int isr_occurred;
```

Comment:
I guess you should init isr_occurred to 0.

Reply:
Oops, I miss it.